### ISSUE TITLE
sounds does not work like the API says

### ISSUE THREAD

**Issue by BlackScorp** [craftyjs/Crafty#220]
http://craftyjs.com/api/Crafty-audio.html#Crafty-audio-add

in example you can see following line:

//adding audio from an object
Crafty.audio.add({
    shoot: ["sounds/shoot.wav",
            "sounds/shoot.mp3",
            "sounds/shoot.ogg"],

```
coin: "sounds/coin.mp3"
```

});

but on Firefox if you call Crafty.audio.play("coin"); you will hear shoot and coin and all other sounds you added at same time. To seperate sounds it works only this way

//adding a single sound
Crafty.audio.add("walk", [
    "sounds/walk.mp3",
    "sounds/walk.ogg",
    "sounds/walk.wav"
]);

//adding a single sound
Crafty.audio.add("coin", [
    "sounds/coin.mp3"
]);

so bug in code? or in API?

**Comment by pengyu** [comment]
I confirm that this is a bug as I see the same thing. See the test case that I made here.

https://github.com/pengyu/Crafty-JS-Examples/blob/master/Crafty/Audio/add/main2.html

sound seem to be not well tested, as I just fixed a bug recently (see the history of the sound source file), which should be very easy to catch and fix, as long as there is a single test case.

sorenbs and Louis Stowasser are the authors of the add function. I think that we should gradually establish a more thorough test policy, otherwise these bugs will jeopardize the reputation of Crafty.
